server/app/services/applications/ApplicationAlreadyInStatus.java

```diff
@@ -0,0 +1,9 @@
+package services.applications;
+
+public class ApplicationAlreadyInStatus extends RuntimeException {
```

Comment:
**suggestion (blocking):** I think the convention for custom exception naming is to end the class name with "Exception".  So, `ApplicationAlreadyInStatusException`.

Reply:
Good catch. I have updated!